.github/workflows/publish-version.yml

```diff
@@ -192,54 +118,85 @@ jobs:
             release/**/*.deb
             release/**/*.rpm
 
-  release:
+  create-release:
     runs-on: ubuntu-latest
-    outputs:
-      new_release_published: ${{ steps.semantic.outputs.new_release_published }}
-      new_release_version: ${{ steps.semantic.outputs.new_release_version }}
     needs:
-      - lint
-      - test
       - build
-
+    outputs:
+      wallet_version: ${{ steps.extract_version.outputs.version }}
+      release_upload_url: ${{ steps.create_release.outputs.upload_url }}
     steps:
       - uses: actions/checkout@v2
         with:
           token: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
 
-      - name: Download distributables
+      - name: Extract version
+        id: extract_version
+        uses: Saionaro/extract-package-version@v1.0.6
+
+      - name: Print version
+        run: echo ${{ steps.extract_version.outputs.version }}
+
+      - name: Create Release
```

Comment:
I might be misunderstanding what's going on here, but won't this create a new release? Isn't that what the semantic-release job did in the create-release workflow? It sounds like this will create two releases.

Reply:
I've changed the semantic-release plugin to not create the release anymore. It creates the changelog, changes the version, pushes the tag, that's it.